python/processors/README.md

```diff
@@ -20,3 +20,12 @@ This example demonstrates the simplest possible variant of a processor. There ar
 It also demonstrates a parser function that outputs operations to create, update, and delete rows.
 
 This processor is part of a full Aptos project, so you can see the processor in context alongside the Move module and frontend.
+
+## Aptos Ambassador Token
+
+This example demonstrates how to track changes on a certain token attribute (e.g., ambassador level) using transactions' events and writesets. This example includes
+
+- Processor: Processes transactions and detects changes on the ambassador level in the events & writesets of the transactions
+- Database: No database operation is demonstrated for the simplicity sake. The changes found are simply printed out in the console.
+
+When an existing token's level changes, the smart contract is programmed to emit an event (e.g., LevelUpdateEvent). So, the level change can be observed both in the transaction's event and writeset redundantly.
```

Comment:
Is the smart contract code public? If so, we can link here so people can see both contract and processor side by side

Reply:
Yes, it's public. I will add a link.